tally/CMakeLists.txt

```diff
@@ -26,6 +26,9 @@ endif (BUILD_SHARED AND ${CMAKE_CXX_COMPILER_ID} STREQUAL Clang)
 add_library(dagtally        SHARED ${SRC_FILES})
 add_library(dagtally-static STATIC ${SRC_FILES})
 
+target_compile_features(dagtally PRIVATE cxx_range_for)
+target_compile_features(dagtally-static PRIVATE cxx_range_for)
```

Comment:
What are these for?

Reply:
This is the most cmakey way of enabling c++std's like c++11

Reply:
Hmm ok. I don't see anything in here that looks like it needs c++11 though

Reply:
Actually I lied, gcc-4.4 doesn't like std::to_string. I guess my main qualm here is that this PR would make it so it's no longer possible to build DAGMC on ACI with the default compiler